src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,99 @@
-export const Pagination = () => {};
+import React from 'react';
+
+type Props = {
+  total: number,
+  perPage: string,
+  currentPage: number,
+  setCurrentPage: (value: number) => void,
+};
+
+export const Pagination:React.FC<Props> = (
+  {
+    total, perPage, currentPage, setCurrentPage,
+  },
+) => {
+  const paginationPageCount = Math.ceil(total / +perPage);
+  const allItems = [];
+
+  for (let i = 1; i <= total; i += 1) {
```

Comment:
why do you need this for because you already have array of items in app.tsx

Reply:
in task we trust says:

"The Pagination should be used with the next props:

<Pagination
  total={42} // total number of items to paginate
  perPage={5} // number of items per page
  currentPage={1} /* optional with 1 by default */
  onPageChange={(page) => { ... }}
/>"